scripts/update-istio-release

```diff
@@ -49,7 +49,7 @@ rm -r charts/rancher-istio/${1}/charts/istio-init
 cat > charts/rancher-istio/${1}/questions.yaml << EOF
 labels:
   rancher.istio.v${1}: ${1}
-rancher_min_version: 2.3.0-rc1
+rancher_min_version: 2.3.4-rc1
```

Comment:
Guessing this will become interactive or not hard coded in future PR ?

Reply:
Yeah, adding an env var with an echo for now though